### ISSUE TITLE
Fix imports to support Components v 2.0.1

### ISSUE THREAD

**Issue by fox1t** [HospitalRun/hospitalrun-frontend#2388]
## 💥 Regression Report

The build is not passing with components v 2.0.1 https://github.com/HospitalRun/hospitalrun-frontend/pull/2387/checks?check_run_id=1094941301 because we changed how components' `scss` files are exported.

## To Reproduce

`npm run build`

## Expected behavior

Build without throwing.

**Comment by fox1t** [comment]
Relates to https://github.com/HospitalRun/components/issues/583 and https://github.com/HospitalRun/components/pull/584

**Comment by CreativeCreate** [comment]
Can I pick this to follow up with my pull-request in @hospitalrun/components https://github.com/HospitalRun/components/pull/593#issue-486944957

Please note: now that there's no need for .scss references in the front-end, the main.scss reference in the front-end must be removed as follows.
```

   ...
   import { Provider } from 'react-redux'
 - import '@hospitalrun/components/scss/main.scss'    // ***to be removed
 - import './index.css'                               // ***include after App import
    import App from './App'
 // ***must import after App to override component styles
 + import './index.css'
  ...

```

**Comment by jackcmeyer** [comment]
Closed via: 3844cd3322f68e12318e90c54161324f1509c8df
